modules/deploy/pages/deployment-option/cloud/managed-connectors/sizing-connectors.adoc

```diff
@@ -0,0 +1,26 @@
+= Sizing Connectors
+:description: How to choose number of tasks to set for a connector.
+:page-aliases: cloud:managed-connectors/task-count.adoc
+:page-cloud: true
+
+== What is a connector task?
+When you set up a connector, its main responsibility is to validate the provided configuration and spawn tasks.
+The actual work is performed by the connector tasks. Setting multiple tasks for a connector allows to pararelize the
+work, resulting in higher throughputs. There are a couple of things that need to be taken into consideration:
+
+- for source connectors, the possibility to increase tasks to achieve higher througput depends entirely on the specific
+connector implementation and configuration, and for many of them only 1 connector task is possible (e.g. for Debezium
+connectors only 1 task can be used).
```

Comment:
```suggestion
- For source connectors, the ability to add tasks to achieve higher throughput depends entirely on the specific
connector implementation and configuration, and for many of them, only a single connector task is allowed (for example, this is the case for Debezium).
```

Reply:
If this is the case, will we provide a table listing the max allowing connector tasks for each source connector?

Reply:
We can add info that when setting number of tasks in the UI is not available, the source connector runs with 1 task only.

Reply:
@Feediver1 I think this is the last remaining suggestion before we can merge.

I added the last sentence based on @tomasz-sadura 's comment.

> When Redpanda Cloud does not offer an option to set the number of tasks, the source connector runs only one task.